apps/precursor-chemicals/translations/src/en/buttons.json

```diff
@@ -10,7 +10,8 @@
   "confirm-and-submit": "Confirm and submit",
   "new-application": "Submit another application",
   "choose-file": "Choose file",
-  "change": "Change", 
+  "change": "Change",
   "remove": "Remove",
-  "add-another-substance": "Add another substance"
-} 
+  "add-another-substance": "Add another substance",
+  "delete": "Remove"
```

Comment:
I noticed there is a `remove` translation on line 14, which probably can be reused? but will require updates on template and js event handlers